src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1289,13 +1289,26 @@ private DetectProperties() {
         new DetectProperty<>(new EnumProperty<>("logging.level.com.synopsys.integration", LogLevel.INFO, LogLevel.class))
             .setInfo("Logging Level", DetectPropertyFromVersion.VERSION_5_3_0)
             .setHelp("The logging level of Detect.",
-                "Detect logging is performed using Spring Boot's default logging setup (Logback). Detect sets the default log message format to \"%d{yyyy-MM-dd HH:mm:ss z} ${LOG_LEVEL_PATTERN:%-6p}[%thread] %clr(---){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}\". You can change your log message format by setting the Spring Boot <i>logging.pattern.console</i> property to a different pattern. Refer to the Spring Boot logging documentation for more details.")
+                "INFO level logging is recommended for normal use to keep the size of the log reasonable. Troubleshooting requires DEBUG or TRACE.<p/>" +
+                    "Detect logging is performed using Spring Boot's default logging setup: Logback. " +
```

Comment:
Change to read: "Detect logging uses the Spring Boot Logback, its default logging setup.

Reply:
Done with an adjustment to address the fact that Logback is an OSS project separate from Spring (something my original sentence didn't make clear.